test/circleci_tests.txt

```diff
@@ -2,6 +2,7 @@
 90_1_prm/test_create_performance_rating_method_baseline_bldg_2.rb
 90_1_prm/test_create_performance_rating_method_baseline_bldg_3
 90_1_prm/test_create_prototype_baseline_building.rb
+90_1_prm/test_create_prototype_baseline_building_res.rb
```

Comment:
Looks like we don't need this test since you've updated the other one. CircleCI won't be able to find it since it hasn't been staged.